tests/files/debian13-calico.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Instance settings
+cloud_image: debian-13
+
+# Kubespray settings
+
+dns_mode: coredns_dual
+kube_asymetric_encryption_algorithm: "RSA-3072"
```

Comment:
HI @tico88612 

`asymetric` seems to be misspelled; it should be `asymmetric`.  
Also, https://github.com/kubernetes-sigs/kubespray/blob/master/tests/files/debian12-calico.yml#L8 appears to have a similar error.

Reply:
All fixed, PTAL.